### ISSUE TITLE
keras implementation 

### ISSUE THREAD

**Issue by tamaraalshekhli** [lorenmt/mtan#32]
thank you for sharing your cod impressive work,
is there any implementation in keras?

**Comment by lorenmt** [comment]
Hi,

There is no Keras version at the current moment, but we would love to assist if the community meets any trouble in reproducibility, and so.

Best,
Sk.

**Comment by tamaraalshekhli** [comment]
@lorenmt, thank you for your reply, i have another questions you mentioned (From my personal experience, designing a better architecture is usually more helpful (and easier) than finding a better task weighting in multi-task learning.) I agree with you but i have problem when im trying to train my MTL model and i found no matter Ive changed the architecture and make it more complicated still the training have the same behavior for the two task i have (good prediction for one and a very bad prediction for the other) is there any explanation for this behavior and is still the same no matter id changed the weight loss value for the both task,

**Comment by lorenmt** [comment]
It shouldn't be this case. I suspect there is a bug in your code. You should check whether the loss is correct, or in the same level of magnitude (all normalize across pixel for example). Also, make sure you have single-task learning (for this bad prediction task) works correctly.

Hope this helps.

**Comment by tamaraalshekhli** [comment]
thank you for your reply, I'm using binary cross entropy for the both task (semantic segmentation) first task is for road extracting and had good prediction in single and multi task the other one is centerline extraction which id tried it for single task the prediction was way better than the multi task, I'm really confused what should i do more

**Comment by lorenmt** [comment]
What exact prediction and loss is for center-line extraction? Could you elaborate more?

**Comment by tamaraalshekhli** [comment]
thank you for your reply and your help really appreciate it , Ive sent you an email with detailed explanation 
i hope you can check it, thank you
